.travis.yml

```diff
@@ -1,52 +1,65 @@
-branches:
-  only:
-   - master
```

Comment:
Do you want this back in? Just noticed that I forgot about it.

Reply:
Nah, you can leave it like that.